chipsec/modules/tools/uefi/uefivar_fuzz.py

```diff
@@ -229,4 +212,4 @@ def run( self, module_argv ):
                 status = self._uefi.delete_EFI_variable(bytestostring(_NAME), str(_GUID))
                 self.logger.log(status)
 
-        return ModuleResult.PASSED
+        return ModuleResult.WARNING
```

Comment:
Most of the tools return passed at the end of the execution. Is there a specific reason why we're not keeping to that?

Reply:
Considering the system could be in an unknown state, even if the module finished, a Warning, further manual evaluation may be needed, seemed more appropriate.